Assets/BossRoom/Scripts/Client/UI/Lobby/LobbyJoiningUI.cs

```diff
@@ -23,6 +24,10 @@ public class LobbyJoiningUI : MonoBehaviour
         IDisposable m_Subscriptions;
         List<LobbyListItemUI> m_LobbyListItems = new List<LobbyListItemUI>();
 
+        // Adding this here until it is available in the Lobby package itself
```

Comment:
what's the jira ticket for this lobby side?
Since they are going to fix this, any reason we're doing this workaround for it here at all?

Reply:
It was still in discussion whether it was something they wanted to expose in the SDK or not, so no Jira ticket yet.
I added this here to simply prevent us from getting the ValidationError, figuring that if we know it will fail, there is no point of actually calling the service.
However, I got some new feedback from the Lobby team about this, I'll wait for the conclusion of that conversation before merging this PR.

Reply:
I ended up removing this and just doing sanitization with the full alphabet instead for this specific PR. If we want to do more and prevent the ValidationError, I think it should be a new ticket, following the conclusion of the aforementioned conversation